packages/block-library/src/media-text/media-container.js

```diff
@@ -67,7 +73,7 @@ class MediaContainer extends Component {
 		const { onSelectMedia, className } = this.props;
 		return (
 			<MediaPlaceholder
-				icon="format-image"
+				icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
				icon={ <BlockIcon icon={ icon } /> }
```